features/setting_constraints/matching_arguments.feature

```diff
@@ -46,6 +46,31 @@ Feature: Matching arguments
       |     expected: (1, anything, /bar/)               |
       |          got: (1, nil, "other")                  |
 
+  Scenario: Using a RSpec matcher
+    Given a file named "rspec_matcher_spec.rb" with:
+      """ruby
+      RSpec.describe "Using a RSpec matcher" do
+        let(:dbl) { double }
+        before { expect(dbl).to receive(:foo).with(contain_exactly(1, 2)) }
```

Comment:
Let's use `a_collection_containing_exactly` rather than `contain_exactly`.  The former is an alias rspec-expectations provides for when you need a noun form to make grammatical sense (it also changes how the matcher is described):

http://rspec.info/documentation/3.2/rspec-expectations/RSpec/Matchers.html#contain_exactly-instance_method